lib/sailor.js

```diff
@@ -261,34 +262,39 @@ class Sailor {
     async fetchMessageBody(message, logger) {
         const { body, headers } = message;
 
-        logger.info('Checking if incoming messages is lightweight...', { headers });
+        logger.debug('Checking if incoming messages is lightweight...');
+        logger.trace({ headers }, 'Checking if incoming messages is lightweight...');
```

Comment:
We discussed that logging data is forbidden to log sensible data at any level. This fix is unacceptable.